cmd/troubleshoot/cli/run.go

```diff
@@ -293,7 +293,8 @@ func loadSpecs(ctx context.Context, args []string, client kubernetes.Interface)
 	}
 
 	// Load additional specs from support bundle URIs
-	if !viper.GetBool("no-uri") {
+	// only when no-uri flag is not set and no URLs are provided in the args
+	if !viper.GetBool("no-uri") && !hasURL(args) {
```

Comment:
Using `support-bundle --dry-run https://my.host.com/spec.yaml spec.yaml`, if `spec.yaml` has a uri, it will also be ignored.

Reply:
Gotcha! I've made the updates. Please let me know how it goes.